crates/web-sys/webidls/enabled/Crypto.webidl

```diff
@@ -17,5 +17,5 @@ interface Crypto {
   readonly attribute SubtleCrypto subtle;
 
   [Throws]
-  ArrayBufferView getRandomValues(ArrayBufferView array);
+  void getRandomValues(ArrayBufferView array);
```

Comment:
Hm it's unfortunate that we need to change this here, could we perhaps change this though so in return position an `ArrayBufferView` is an `Object` but in argument position it's either an object or a u8 array?

Reply:
It was little awkward, but I got the parser to do just that: consider it a union for arguments, assume it to be an JS `Object` when returned.